radix-engine/src/kernel/kernel.rs

```diff
@@ -52,6 +53,21 @@ impl KernelBoot {
     pub fn babylon() -> Self {
         Self::V1
     }
+
+    pub fn cuttlefish() -> Self {
+        Self::V2(AlwaysVisibleGlobalNodesVersion::V2)
+    }
+
+    pub fn always_visible_global_nodes_version(&self) -> AlwaysVisibleGlobalNodesVersion {
```

Comment:
I wonder if we can make this not pub?

Reply:
Yup, but why not keep it public?

Reply:
Very minor, it just feels like an internal detail that other things shouldn't need to care about :)